spec/controllers/v0/chatbot/claim_status_controller_spec.rb

```diff
@@ -163,6 +163,7 @@
 
     context 'authorized' do
       before do
+        allow(Flipper).to receive(:enabled?).and_call_original
         @mock_cxi_reporting_service = instance_double(Chatbot::ReportToCxi)
         allow(@mock_cxi_reporting_service).to receive(:report_to_cxi)
```

Comment:
This spec is only at 85% code coverage. Please update spec to resolve code coverage issues.